src/app/twinclass/twin-class-dialog.tsx

```diff
@@ -193,86 +196,101 @@ export function TwinClassDialog({
                 <DialogDescription>
 
                 </DialogDescription>
+                <Separator className="border-t-2 border-t-[#5e5e5e]"/>
```

Comment:
Put separators in header and footer respectively, ON by default, with option to disable it
Go through other dialogs in the project and make sure they work correctly (you may need to put autofocus there too, later we'll think about a more robust solution)